### PR TITLE
consolidate all duplicate CVRs

### LINKED ISSUE #692: Create a common test suite for shared CVRs
See comment https://github.com/BrightSpots/rcv/pull/650#discussion_r1224591566

In the `test_data` directory, create a `common` directory in which multiple tests can share a single CVR, to avoid unnecessary duplication of tens of thousands of lines of CVR data.

### PR DESCRIPTION
**Pull request by artoonie**
closes #692 

Note: one of the original goals was to specifically get rid of the duplicate files in #650, but I opted to skip that: while most files are the same, one test removes the Precinct file, and the other doesn't, meaning the test sets aren't identical. I'm sure we could solve that with some added complexity, but I'm not sure that complexity is worth it.

### CODE REVIEW

**Review by HEdingfield — CHANGES_REQUESTED** [review]
Placing a hold on this until we get the parent branch merged. Will review once that's done.

File: src/test/resources/network/brightspots/rcv/test_data/minneapolis_multi_seat_threshold/minneapolis_multi_seat_threshold_config.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   },
   "cvrFileSources": [
     {
-    "filePath" : "minneapolis_multi_seat_threshold_cvr.xlsx",
+    "filePath" : "../_shared/minneapolis_multi_seat_cvr.xlsx",
```
Comment (HEdingfield) [inline src/test/resources/network/brightspots/rcv/test_data/minneapolis_multi_seat_threshold/minneapolis_multi_seat_threshold_config.json:14]: It seems like this one might not actually be shared between any other tests? Are we pre-emptively moving CVRs into the `_shared` folder, or was this a mistake?

File: src/test/resources/network/brightspots/rcv/test_data/skip_to_next_test/skip_to_next_test_config.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "generateCdfJson" : false
   },
   "cvrFileSources" : [ {
-    "filePath" : "skip_to_next_test_cvr.xlsx",
+    "filePath" : "../_shared/skipped_ranking_cvr.xlsx",
```
Comment (HEdingfield) [inline src/test/resources/network/brightspots/rcv/test_data/skip_to_next_test/skip_to_next_test_config.json:13]: Same question on this one -- seems like this isn't used in any other tests.

File: src/test/resources/network/brightspots/rcv/test_data/test_set_2_overvote_skip_to_next/test_set_2_overvote_skip_to_next_cvr.xlsx
Comment (HEdingfield) [inline src/test/resources/network/brightspots/rcv/test_data/test_set_2_overvote_skip_to_next/test_set_2_overvote_skip_to_next_cvr.xlsx:1]: Kind of weird that this wasn't used at all... was this just junk all along, or could it possibly serve some value?

File: src/test/resources/network/brightspots/rcv/test_data/test_set_multi_winner_fractional_threshold/test_set_multi_winner_fractional_threshold_cvr.xlsx
Comment (HEdingfield) [inline src/test/resources/network/brightspots/rcv/test_data/test_set_multi_winner_fractional_threshold/test_set_multi_winner_fractional_threshold_cvr.xlsx:1]: Same question on this... wondering why this was around if it apparently wasn't being used?

**Review by HEdingfield — COMMENTED** [review]
This is great clean-up! Just a few questions before merging.

File: src/test/resources/network/brightspots/rcv/test_data/test_set_multi_winner_fractional_threshold/test_set_multi_winner_fractional_threshold_cvr.xlsx
Comment (artoonie) [reply src/test/resources/network/brightspots/rcv/test_data/test_set_multi_winner_fractional_threshold/test_set_multi_winner_fractional_threshold_cvr.xlsx:1]: Hummm -- for all four comments here, the two *unused* files were duplicates, which is why the two *unshared* CVRs were moved to `_shared`.

In particular, `../_shared/skipped_ranking_cvr.xlsx` == `test_set_2_overvote_skip_to_next_cvr.xlsx` and 
`../_shared/minneapolis_multi_seat_cvr.xlsx ` == `../minneapolis_multi_seat_threshold/minneapolis_multi_seat_cvr.xlsx`.

It seems as though the xlsx files are indeed never used -- that they've been converted to CDF already. So I removed the duplicate files and placed them in shared, but one of the duplicate files were never used.

I have two options:
1. Delete the CDF CVR and point to the shared XLSX CVR (which seems to be the ground truth anyway), or
2. Leave these XLSX files deleted and move the two files in `_shared` out of the shared directory.

I think (1) is cleaner -- let's use the source files? Unless there was some reason why they weren't being used in the first place.

**Review by HEdingfield — APPROVED** [review]
Awesome job! Left a few last nit comments, but pre-approving.

**Pull request merged**